features/draft/iframe-sandbox.yml

```diff
@@ -0,0 +1,31 @@
+draft_on: 2024-07-19
+# Revisit publishing this when the following are done:
+# https://github.com/mdn/browser-compat-data/pull/23862
+# https://github.com/mdn/browser-compat-data/pull/23858
+# https://github.com/mdn/browser-compat-data/pull/23861
+# https://github.com/mdn/browser-compat-data/pull/23864
+name: Iframe sandbox
```

Comment:
I think capital-I Iframe is weird and maybe exceptional. If I'm alone on this, I won't worry about it though.

Reply:
It's a bit like "Canvas reset()" but I agree it's weirder, because "Iframe" doesn't look as much like a word.

https://github.com/web-platform-dx/web-features/issues/1389 is also in this stye, I'd be happy to change them to another style, like "sandbox for <iframe>".

Reply:
Having a look around Stackoverflow and the like, I think "Sandboxed iframe" is most common, followed closely by "Sandbox iframe." Credentialless is newer and less definitive (with the oddly ungrammatical "Iframe credentialless" appearing), but "Credentialless iframe" is a plausible option too. "\_ for <iframe>" would be fine too, though a touch inventive on our part.

Reply:
> So... WDYT about "Sandboxed iframes", "Credentialless iframes", and "srcdoc"?

Yes, let's do it. I think this is all in the spirit of prioritizing names that developers actually use.

Reply:
```suggestion
name: Sandboxed iframes
```

Reply:
I've sent https://github.com/web-platform-dx/web-features/pull/1504 too.